docs/sources/tempo/operations/user-configurable-overrides.md

```diff
@@ -107,15 +107,15 @@ POST /api/overrides
 
 Update the overrides with the given payload. Note this will overwrite any existing overrides.
 
-Example: `curl -X POST -v -H "X-Scope-OrgID: 3" -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides --data "{}"
+Example: `curl -X POST -v -H "X-Scope-OrgID: 3" -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides --data "{}"`
```

Comment:
Do we want this example to also be in a code block like the one at the end of the page? I was going to type it as a suggestion but the suggestions go in codeblocks and don't play nicely together.

Reply:
Thank you.